Tests/TensorFlowTests/OperatorTests/DatasetTests.swift

```diff
@@ -111,7 +111,7 @@ final class DatasetTests: XCTestCase {
         let shuffled = dataset.shuffled(sampleCount: 5, randomSeed: 42)
         XCTAssertEqual([0, 4, 1, 3, 2], shuffled.map { $0.scalar! })
         let evens = shuffled.map { Tensor($0 % 2) .== Tensor(0) }
-        XCTAssertEqual([true, true, false, false, true], evens.map { $0.scalar! })
+        XCTAssertEqual([true, true, false, false, true], evens.map { $0.scalar! } )
```

Comment:
```suggestion
        XCTAssertEqual(evens.map { $0.scalar! }, [true, true, false, false, true])
```

Reply:
@rxwei I will make these changes, but when i tried to locally build it before reverting these changes, This didn't pass either.